scripts/scripts.js

```diff
@@ -1076,9 +1079,63 @@ async function loadLazy(doc) {
   }
   sampleRUM('lazy');
   await headerloaded;
-  sampleRUM.convert(document.querySelectorAll('a[href^="https://www.bamboohr.com/signup"]'), 'signup');
-  sampleRUM.convert(document.querySelectorAll('a[href^="https://www.bamboohr.com/pl-pages/demo-request/"]'), 'demo-request');
-  sampleRUM.convert(document.querySelectorAll('a[data-modal="#pricing-modal"]'), 'pricing-quote');
+  // eslint-disable-next-line no-use-before-define
+  registerConversionListeners(document);
+}
+
+/**
+ * Returns the label used for tracking link clicks
+ * @param {Element} element link element
+ * @returns link label used for tracking converstion
+ */
+function getLinkLabel(element) {
+  return element.title ? toClassName(element.title) : toClassName(element.innerHTML);
+}
+
+/**
+ * Registers conversion listeners according to the metadata configured in the document.
+ * @param {Element} parent element where to find potential event conversion sources
+ * @param {string} path fragment path when the parent element is coming from a fragment
+ */
+export async function registerConversionListeners(parent, path) {
+
+  const conversionElements = getMetadata('conversion-element')?.split(',').map((ce) => toClassName(ce.trim()));
+  if (conversionElements && Array.isArray(conversionElements)) {
```

Comment:
```suggestion
  if (Array.isArray(conversionElements)) {
```

`Array.isArray(undefined) === false`

Reply:
Done